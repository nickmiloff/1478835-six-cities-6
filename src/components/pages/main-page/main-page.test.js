import React from 'react';
import {render} from '@testing-library/react';
import Test from '../../test/test';

import MainPage from './main-page';

describe(`MainPage component test`, () => {
  it(`'MainPage component' should render correctly`, () => {
    const {container} = render(
        <Test>
          <MainPage />
        </Test>
    );

    expect(container).toMatchSnapshot();
  });
});
