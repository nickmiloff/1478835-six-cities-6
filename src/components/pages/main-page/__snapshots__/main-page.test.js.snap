// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage component test 'MainPage component' should render correctly 1`] = `
<div>
  <div
    class="page page--gray page--main"
  >
    <header
      class="header"
    >
      <div
        class="container"
      >
        <div
          class="header__wrapper"
        >
          <div
            class="header__left"
          >
            <a
              class="header__logo-link header__logo-link--active"
              href="/"
            >
              <img
                alt="6 cities logo"
                class="header__logo"
                height="41"
                src="img/logo.svg"
                width="81"
              />
            </a>
          </div>
          <nav
            class="header__nav"
          >
            <ul
              class="header__nav-list"
            >
              <li
                class="header__nav-item user"
              >
                <a
                  class="header__nav-link header__nav-link--profile"
                  href="/login"
                >
                  <div
                    class="header__avatar-wrapper user__avatar-wrapper"
                  />
                  <span
                    class="header__login"
                  >
                    Sign in
                  </span>
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </div>
    </header>
    <main
      class="page__main page__main--index page__main--index-empty"
    >
      <h1
        class="visually-hidden"
      >
        Cities
      </h1>
      <div
        class="tabs"
      >
        <section
          class="locations container"
        >
          <ul
            class="locations__list tabs__list"
          >
            <li
              class="locations__item"
            >
              <a
                class="locations__item-link tabs__item tabs__item--active"
                data-testid="tab-link"
              >
                <span>
                  Paris
                </span>
              </a>
            </li>
            <li
              class="locations__item"
            >
              <a
                class="locations__item-link tabs__item"
                data-testid="tab-link"
              >
                <span>
                  Cologne
                </span>
              </a>
            </li>
            <li
              class="locations__item"
            >
              <a
                class="locations__item-link tabs__item"
                data-testid="tab-link"
              >
                <span>
                  Brussels
                </span>
              </a>
            </li>
            <li
              class="locations__item"
            >
              <a
                class="locations__item-link tabs__item"
                data-testid="tab-link"
              >
                <span>
                  Amsterdam
                </span>
              </a>
            </li>
            <li
              class="locations__item"
            >
              <a
                class="locations__item-link tabs__item"
                data-testid="tab-link"
              >
                <span>
                  Hamburg
                </span>
              </a>
            </li>
            <li
              class="locations__item"
            >
              <a
                class="locations__item-link tabs__item"
                data-testid="tab-link"
              >
                <span>
                  Dusseldorf
                </span>
              </a>
            </li>
          </ul>
        </section>
      </div>
      <div
        class="cities"
      >
        <div
          class="cities__places-container container cities__places-container--empty"
        >
          <section
            class="cities__no-places"
          >
            <div
              class="cities__status-wrapper tabs__content"
            >
              <b
                class="cities__status"
              >
                No places to stay available
              </b>
              <p
                class="cities__status-description"
              >
                We could not find any property available at the moment in 
                Paris
              </p>
            </div>
          </section>
          <div
            class="cities__right-section"
          />
        </div>
      </div>
    </main>
  </div>
</div>
`;
